urdfenvs/sensors/obstacleSensor.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
```

Comment:
This file should not be needed as it is a copy of the file `urdfenvs/sensors/obstacle_sensor.py`
The naming was changed in #61.